pandas/tests/util/test_util.py

```diff
@@ -37,22 +35,6 @@ def test_numpy_err_state_is_default():
     assert np.geterr() == expected
 
 
-@pytest.mark.parametrize("func,expected", [
-    # Case where the func does not have default kwargs.
-    (validate_kwargs, (["fname", "kwargs", "compat_args"],
-                       ["fname", "kwargs", "compat_args"])),
-
-    # Case where the func does have default kwargs.
-    (deprecate_kwarg, (["old_arg_name", "new_arg_name",
-                        "mapping=None", "stacklevel=2"],
-                       ["old_arg_name", "new_arg_name",
-                        "mapping", "stacklevel"]))
-])
-def test_make_signature(func, expected):
```

Comment:
It can be noted that ``validate_kwargs`` and ``deprecate_kwarg`` have their own test modules, so removing here it not an issue.